frontend/lib/account-settings/route-info.ts

```diff
@@ -4,7 +4,8 @@ export function createAccountSettingsRouteInfo(prefix: string) {
   return {
     [ROUTE_PREFIX]: prefix,
     home: prefix,
-    name: `${prefix}/name`,
+    legalname: `${prefix}/legalname`,
+    preferredname: `${prefix}/preferredname`,
```

Comment:
Yes, I remember that conversation - I just don't know if I got around to implementing it! Let's talk about this tomorrow - this will have implications for the whole `SessionUpdatingFormSubmitter` thing about whether there should be two or one.